doc/osh.1

```diff
@@ -51,11 +51,9 @@ OSH_CRASH_DUMP_DIR
 On startup,
```

Comment:
Since there is only one file, how about:

```mdoc
The interactive shell sources only
.Pa ~/.config/oil/oshrc
on startup.
```

Reply:
Done.